### PR TITLE
Start of Docker images projects

### PR DESCRIPTION
**Pull request by glicht**
Base Docker images

### CODE REVIEW

File: docker/python3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM python:3.7.1-alpine3.8
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+RUN addgroup -g 4000 demisto \
+  && adduser -u 4000 -G demisto -D demisto -s /bin/sh
```
Comment (dorsha) [inline docker/python3/Dockerfile]: it won't be always demisto demisto, it can be also socserver and rsa

File: docker/python3-deb/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# Basic linux utilities
+RUN apt-get update \
+  && apt-get install -y \
+    procps
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+RUN groupadd --gid 4000 demisto \
```
Comment (dorsha) [inline docker/python3-deb/Dockerfile]: same here

File: docker/dockerbuild/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FROM circleci/python:2.7.15-stretch
```
Comment (meirwah) [inline docker/dockerbuild/Dockerfile]: why not `slim` version of stretch here too?

File: docker/python-deb/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# Basic linux utilities
+RUN apt-get update \
+  && apt-get install -y \
+    procps
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+RUN groupadd --gid 4000 demisto \
```
Comment (meirwah) [inline docker/python-deb/Dockerfile]: did you test these docker images work well in demisto ? on various OS's ?

File: docker/dockerbuild/Dockerfile (hunk 3 above)
Comment (glicht) [reply docker/dockerbuild/Dockerfile]: This is from circleci. It is an image to use for building our images (dev only). It is not an image we will be shipping. Circle don't have a slim version from what I saw.

File: docker/python-deb/Dockerfile (hunk 4 above)
Comment (glicht) [reply docker/python-deb/Dockerfile]: Only tested locally (mac). Need to discuss how we want to test them. I am open for suggestions. We can take this offline.

**Pull request merged**